src/segmantic/prepro/labels.py

```diff
@@ -33,19 +57,40 @@ def random_color(l, max_label):
             hue = min(hue, 1.0)
             return colorsys.hls_to_rgb(hue, 0.5, 1.0)
 
+        tissue_color_map = lambda n: random_color(tissue_label_map[n], num_tissues)
+
+    with open(tissue_list_file_name, "w") as f:
         print("V7", file=f)
         print("N%d" % num_tissues, file=f)
-        for label in range(1, num_tissues+1):
-            r,g,b = random_color(label, num_tissues)
-            print("C%.2f %.2f %.2f %.2f %s" % (r,g,b, 0.5, label_tissue_map[label]), file=f)
+        for label in range(1, num_tissues + 1):
+            name = label_tissue_map[label]
+            r, g, b = tissue_color_map(name)
+            print(
+                "C%.2f %.2f %.2f %.2f %s" % (r, g, b, 0.5, name),
+                file=f,
+            )
 
 
-def load_tissue_list(file_name) -> dict:
-    tissues = { 0: "Background" }
+def load_tissue_list(file_name) -> Dict[str, int]:
```

Comment:
TIP: i would do all interfaces to ``load/dump`` with streams instead of file paths (could eventualy  have one wrapper with filepaths). Notice e.g. libraries like ``json.load, json.dump`` or ``yaml.load, yaml.dump`` ... 

e.g.  if you connect this library to a restAPI, you get all in strings instead of files  so you could use the python-equivalent to C++ ``stringstream`` which is [io.StringIO](https://docs.python.org/3/library/io.html#io.StringIO)